pyeudiw/tests/settings.py

```diff
@@ -338,7 +340,7 @@ def base64url_to_int(val):
             "module": "pyeudiw.trust.handler.x509",
             "class": "X509Handler",
             "config": {
-                "client_id": f"{BASE_URL}/OpenID4VP",
+                "client_id": f"x509_san_dns:{BASE_URL}/{BACKEND_NAME}",
```

Comment:
```suggestion
                "client_id": f"x509_san_dns:...",
```

no, the dns name is not a URL scheme and it cannot contain therefore any webpaths